### ISSUE TITLE
mumble & ice gone in 21?

### ISSUE THREAD

**Issue by nambrosch** [fedora-infra/fedora-packages#138]
after upgrading from 20->21, i noticed that the packages murmur, ice, and ice-php are no longer in the fedora repo.. is this intended?

**Comment by pypingou** [comment]
While this is the wrong place to ask for such question, it seems that ice was retired for F21+:
https://admin.fedoraproject.org/pkgdb/package/ice/ (I suspect ice-php was a sub-package)

As for murmur, I do not even see it in pkgdb nor in packages (which is the application this github project is about): https://apps.fedoraproject.org/packages/s/murmur.

I guess such question would be better asked on http://ask.fedoraproject.org/ or on the user mailing list.

**Comment by nambrosch** [comment]
I realized it was the wrong place to post after opening the issue, sorry about that.

After much searching i found that the retiring of ice+mumble from the fedora repo was a misunderstanding and they're working to get it back in.  In case anyone else stumbles across this post, I found two quasi-official builds that are working for me.

http://koji.fedoraproject.org/koji/buildinfo?buildID=562386
http://koji.fedoraproject.org/koji/buildinfo?buildID=580253
